sky/utils/common_utils.py

```diff
@@ -61,11 +61,17 @@ def get_usage_run_id() -> str:
     return _usage_run_id
 
 
-def get_user_hash() -> str:
+def get_user_hash(force_fresh_hash: bool = False) -> str:
```

Comment:
any reason for adding this `force_fresh_hash`? It seems we don't use it elsewhere.

Reply:
It is used in `kubernetes.ssh_key_secret_field_name` to override the user_hash set for the controller through envvars